src/P5/scripts/readme.md

```diff
@@ -0,0 +1,118 @@
+# P5/scripts - README
```

Comment:
I think having a sphinx documentation makes more sense than writing all these readmes. Make sure you can port them later on